internal/provider/kubernetes/controller.go

```diff
@@ -1155,6 +1180,7 @@ func (r *gatewayAPIReconciler) subscribeAndUpdateStatus(ctx context.Context) {
 
 // watchResources watches gateway api resources.
 func (r *gatewayAPIReconciler) watchResources(ctx context.Context, mgr manager.Manager, c controller.Controller) error {
+	log.SetLogger(zap.New(zap.WriteTo(os.Stderr), zap.UseDevMode(true)))
```

Comment:
is this needed ? can we use `r.log` directly ?

Reply:
removed it, this is not required.